tf2_py/CMakeLists.txt

```diff
@@ -101,6 +102,10 @@ else()
   set_target_properties(tf2_py PROPERTIES COMPILE_FLAGS "-g -Wno-missing-field-initializers")
   set_target_properties(tf2_py PROPERTIES OUTPUT_NAME tf2 PREFIX "_" SUFFIX ".so")
 endif()
+if(APPLE)
+  set_target_properties(tf2_py PROPERTIES
+                        LINK_FLAGS "-undefined dynamic_lookup")
+endif()
```

Comment:
Maybe put this in the else above?

Reply:
But then it would apply to Linux too which is not what we want, is it?

Reply:
Ok, done.